tests/public-install/variables.tf

```diff
@@ -1,14 +1,44 @@
+variable "aws_role_arn" {
+  type        = string
+  description = "The AWS Role ARN to assume for this module."
+}
+
 variable "domain_name" {
   type        = string
   description = "Domain for creating the Terraform Enterprise subdomain on."
 }
 
-variable "license_path" {
+variable "external_bootstrap_bucket" {
   type        = string
-  description = "File path to Replicated license file"
+  description = "The name of the S3 bucket for bootstrap artifacts."
+  default     = null
 }
 
 variable "acm_certificate_arn" {
   type        = string
   description = "The ARN of an existing ACM certificate."
 }
+
+variable "network_id" {
+  default     = ""
```

Comment:
Since the test case is hard-coded not to create a VPC, the network inputs should be required since the deployment will not proceed without them.

Reply:
I'm not sure how to proceed with this, as the only reason the test doesn't create a VPC is because of how we manage the base-infra to already have a VPC, but originally the tests could create their own. I'll remove it. but this is making the tests less flexible than they were as examples.